amy/fiscal/forms.py

```diff
@@ -257,16 +280,19 @@ def __init__(self, *args, **kwargs):
         # set up layout objects for the helpers - they're identical except for
         # visibility of the delete checkbox
         self.helper.layout = self.helper.build_default_layout(self)
-        self.helper_empty_form.layout = self.helper.build_default_layout(self)
         self.helper.layout.append(Field("id"))
         self.helper.layout.append(Field("DELETE"))  # visible; formset adds it
+        self.helper_empty_form.layout = self.helper.build_default_layout(self)
         self.helper_empty_form.layout.append(Field("id"))
         self.helper_empty_form.layout.append(
             Div(Field("DELETE"), css_class="d-none")  # hidden
         )
+        # remove EDITABLE checkbox from empty helper form
+        pos_index = self.helper_empty_form.layout.fields.index("EDITABLE")
+        self.helper_empty_form.layout.pop(pos_index)
```

Comment:
Maybe one test (or part of one test) that asserts self.helper_empty_form does not include the EDITABLE field.

Reply:
I moved Member-related tests to a separate file and added layout tests for MemberForm and MembershipTaskForm.